.semaphore.yml

```diff
@@ -15,9 +15,8 @@ project_settings:
       - docker-compose -f docker-compose.test.yml build
       - docker push $DOCKER_IMAGE:$BRANCH_TAG
 
-    # In order to run proto, the `.pronto.yml` needs to be set
     unit_tests:
-      - if ([ $BRANCH_NAME != 'master' ] && [ $BRANCH_NAME != 'development' ]); then (echo "Running pronto"; docker-compose -f docker-compose.test.yml run test bundle exec pronto run -f bitbucket_pr -c origin/development); else (echo "Escaping pronto"); fi || true
+      - if ([ $BRANCH_NAME != 'master' ] && [ $BRANCH_NAME != 'development' ]); then (echo "Running Danger"; docker-compose -f docker-compose.test.yml run test bundle exec danger); else (echo "Escaping Danger"); fi || true
```

Comment:
do we need to add the `DANGER_GITHUB_API_TOKEN` on Semaphore?

Reply:
In my findings No,  adding it to `docker-compose` will work fine. @rafayetn am I right?

Reply:
As `semaphore` is running `unit tests` within docker container and as we specified `DANGER_GITHUB_API_TOKEN` in `docker-compose` it should be enough I think.

Reply:
Does it mean the TOKEN could be public? as we set it in the docker-compose, it will appear on Github? Should we consider about that case?

Reply:
It should always be secret as you said. We updated it in the Readme.